packages/cli/src/commands/contract/call.ts

```diff
@@ -1,49 +1,87 @@
 import { Command, Flags } from "@oclif/core";
-import { execSync } from "node:child_process";
 import path = require("node:path");
-import { getSwankyConfig, resolveNetworkUrl } from "@astar-network/swanky-core";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+import { ContractPromise } from "@polkadot/api-contract";
+import {
+  AccountData,
+  ChainAccount,
+  ChainApi,
+  decrypt,
+  Encrypted,
+  getSwankyConfig,
+  resolveNetworkUrl,
+  Spinner,
+  AbiType as Abi,
+} from "@astar-network/swanky-core";
+import inquirer from "inquirer";
+import chalk = require("chalk");
+import { readJSON } from "fs-extra";
 
 export class CallContract extends Command {
   static description = "Call a method on a smart contract";
 
+  static args = [
+    {
+      name: "contractName",
+      required: true,
+      description: "Name of the contract to call",
+    },
+    {
+      name: "messageType",
+      required: true,
+      description: "Is the call a query or tx type",
+      options: ["q", "query", "tx"],
+    },
+  ];
+
   static flags = {
-    args: Flags.string({
+    params: Flags.string({
       required: false,
-      char: "a",
+      description: "Arguments supplied to the message",
+      multiple: true,
+      default: [],
+      char: "p",
     }),
-    contractName: Flags.string({
+    account: Flags.string({
```

Comment:
I think for query call, we can use Alice by default without requiring users to input.